doc/main/tbb_userguide/design_patterns/Lazy_Initialization.rst

```diff
@@ -0,0 +1,108 @@
+.. _Lazy_Initialization:
+
+Lazy Initialization
+==================
+
+
+.. container:: section
+
+
+   .. rubric:: Problem
+      :class: sectiontitle
+
+   Delay the creation of an object (potentially expensive) until it is accessed.
+   In parallel programming initialization also must be guarded against race conditions.
+
+
+.. container:: section
+
+
+   .. rubric:: Context
+      :class: sectiontitle
+
+   The cost of operations that take place during the initialization
+   of the object may be considerably high. In that case, the object
+   should be initialized only when it is needed. Lazy initialization
+   is the common tactic that allows implementing such approach.
+
+
+.. container:: section
+
+
+   .. rubric:: Solution
+      :class: sectiontitle
+
+   Using ``oneapi::tbb::collaborative_call_once`` with ``oneapi::tbb::collaborative_once_flag``
+   helps to implement thread-safe lazy initialization for a user object.
+
+
+   In addition, ``collaborative_call_once`` allows other thread blocked on
+   the same ``collaborative_once_flag`` to join other |short_name|
+   parallel constructions called within the initializing function.
+
+
+.. container:: section
+
+
+   .. rubric:: Example
+      :class: sectiontitle
+
+   The example presented here illustrate implementation of "lazy initialization" for Fibonacci numbers calculation.
```

Comment:
```suggestion
   This example illustrates the implementation of "lazy initialization" for the calculation of the Fibonacci numbers.
```

Reply:
Why is lazy initialization in quotes here but not in the cases above?

Reply:
Removed quotes.